reframe/frontend/autodetect.py

```diff
@@ -136,7 +136,8 @@ def _is_part_local(part):
             part.launcher_type.registered_name == 'local')
 
 
-def _remote_detect(part):
+def _remote_detect(part, job_options=None):
+    job_options = [] if job_options is None else job_options
```

Comment:
I would avoid having another default here. This is an internal function after all:

```suggestion
def _remote_detect(part, job_options):
```

Reply:
Maybe also call the argument `cli_job_options` to match the name we use in the scheduler backends.

Reply:
Ah, I didn't notice the second comment. Do you mean in `_remote_detection` only or in both `_remote_detection` and `detect_topology`?